app/src/androidTest/java/com/pajato/android/gamechat/ApplicationTest.java

```diff
@@ -81,33 +81,9 @@
                 .check(matches(isDisplayed()));
     }
 
-    /** Ensure that switching between two different game panels works properly */
-    @Test public void testGameSwitcher() {
-        // Move to the game pane and ensure that we are starting on the settings panel.
-        onView(withId(R.id.toolbar_game_icon))
-                .perform(click());
-        onView(withId(R.id.settings_panel))
-                .check(matches(isDisplayed()));
-        // Open the action bar overflow menu, then initiate a new TTT game. Ensure it functions.
-        openActionBarOverflowOrOptionsMenu(InstrumentationRegistry.getTargetContext());
-        onView(withText(R.string.new_game_ttt))
-                .check(matches(isDisplayed()))
-                .perform(click());
-        onView(withId(R.id.board))
-                .check(matches(isDisplayed()));
-        // Reopen the action bar overflow menu, then ensure returning to the settings pane works.
-        openActionBarOverflowOrOptionsMenu(InstrumentationRegistry.getTargetContext());
-        onView(withText(R.string.new_game_settings))
-                .check(matches(isDisplayed()))
-                .perform(click());
-        onView(withId(R.id.settings_panel))
-                .check(matches(isDisplayed()));
-    }
-
     /** Ensure that the hamburger menu is dismissed on a back press. */
```

Comment:
In general, with our preferred style of short test methods, we can probably get away with either the javadoc comment or the code comment.  I don't think we need both to have a clue about what the code is doing and why.  I have no preference on which one we use.